common/src/main/java/com/revenuecat/purchases/common/Backend.kt

```diff
@@ -48,6 +53,12 @@ class Backend(
     @get:Synchronized @set:Synchronized
     @Volatile var offeringsCallbacks = mutableMapOf<String, MutableList<OfferingsCallback>>()
 
+    @get:Synchronized @set:Synchronized
+    @Volatile var createAliasCallbacks = mutableMapOf<CallbackCacheKey, MutableList<CreateAliasCallback>>()
+
+    @get:Synchronized @set:Synchronized
+    @Volatile var identifyCallbacks = mutableMapOf<CallbackCacheKey, MutableList<IdentifyCallback>>()
+
```

Comment:
it's fine for now, but we really should try to make this a common thing, which we can reuse easily. Perhaps at the HTTPClient level

Reply:
Same thoughts I had